.github/workflows/tests.yml

```diff
@@ -1,33 +1,6 @@
 name: tests
 on: [pull_request]
 jobs:
-  base-tests:
```

Comment:
I guess this will stop after we merge huh? Odd that it shows in this PR

Reply:
I think so too.